configure.ac

```diff
@@ -3538,8 +3539,8 @@ AC_ARG_WITH([libmysql],
     else
       if test -f "$withval" && test -x "$withval"; then
         with_mysql_config="$withval"
-      else if test -x "$withval/bin/mysql_config"; then
-        with_mysql_config="$withval/bin/mysql_config"
```

Comment:
I'd prefer if we checked for *mariadb_config* in addition to *mysql_config*. I think we should check for *mariadb_config* first, to prefer the newer system.